### PR TITLE
Create support.md for .net repo

### PR DESCRIPTION
**Pull request by elraikhm**

### CODE REVIEW

File: support.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+- Chat with other community members on [gitter](https://gitter.im/Azure/azure-sdk-for-net?source=orgpage)
+- Ask a question on [Twitter](https://twitter.com/AzureSDK?s=20)
+- Ask a question at [Microsoft Q&A](https://docs.microsoft.com/en-us/answers/products/azure?WT.mc_id=Portal-Microsoft_Azure_Support&product=all)
+- Ask a question at [Microsoft Tech Community](https://techcommunity.microsoft.com/t5/azure/ct-p/Azure)
+- Search for similar issues in [our GitHub repository](https://github.com/Azure/azure-sdk-for-net/issues)
+
+### Security bugs
+Security issues and bugs should be reported privately, via email, to the Microsoft Security Response Center(secure@microsoft.com). 
+You should receive a response within 24 hours. 
+Further information, including the MSRC PGP key, can be found in the [Security TechCenter](https://www.microsoft.com/en-us/msrc/faqs-report-an-issue?rtc=1)
```
Comment (jsquire) [inline support.md:21]:
```suggestion
Further information, including the MSRC PGP key, can be found in the [Security TechCenter](https://www.microsoft.com/msrc/faqs-report-an-issue?rtc=1)
```

File: support.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+Customers with a [support plan](https://azure.microsoft.com/en-us/support/options/) can open an Azure Support ticket [here](https://azure.microsoft.com/en-us/support/create-ticket/).
+**We recommend this option if your problem requires immediate attention.** 
+
+### Community resources
+- Ask a question on [StackOverflow](https://stackoverflow.com/questions/tagged/azure-sdk-.net) and tag it with azure-sdk-.net
+- Share or upvote feature requests on [Feedback Page](https://feedback.azure.com/forums/34192--general-feedback).
+- Take a look at the [Azure SDK blog](https://devblogs.microsoft.com/azure-sdk/).
+- Chat with other community members on [gitter](https://gitter.im/Azure/azure-sdk-for-net?source=orgpage)
+- Ask a question on [Twitter](https://twitter.com/AzureSDK?s=20)
+- Ask a question at [Microsoft Q&A](https://docs.microsoft.com/en-us/answers/products/azure?WT.mc_id=Portal-Microsoft_Azure_Support&product=all)
```
Comment (jsquire) [inline support.md:14]:
```suggestion
- Ask a question at [Microsoft Q&A](https://docs.microsoft.com/answers/products/azure?WT.mc_id=Portal-Microsoft_Azure_Support&product=all)
```

File: support.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+Support
+
+## How to file issues and get help  
+
+Customers with a [support plan](https://azure.microsoft.com/en-us/support/options/) can open an Azure Support ticket [here](https://azure.microsoft.com/en-us/support/create-ticket/).
```
Comment (jsquire) [inline support.md:5]:
```suggestion
Customers with a [support plan](https://azure.microsoft.com/en-us/support/options/) can open an Azure Support ticket [here](https://azure.microsoft.com/support/create-ticket/).
```

File: support.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+Support
```
Comment (jsquire) [inline support.md:1]:
```suggestion
# Support
```

File: support.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+Customers with a [support plan](https://azure.microsoft.com/en-us/support/options/) can open an Azure Support ticket [here](https://azure.microsoft.com/en-us/support/create-ticket/).
+**We recommend this option if your problem requires immediate attention.** 
+
+### Community resources
+- Ask a question on [StackOverflow](https://stackoverflow.com/questions/tagged/azure-sdk-.net) and tag it with azure-sdk-.net
+- Share or upvote feature requests on [Feedback Page](https://feedback.azure.com/forums/34192--general-feedback).
+- Take a look at the [Azure SDK blog](https://devblogs.microsoft.com/azure-sdk/).
+- Chat with other community members on [gitter](https://gitter.im/Azure/azure-sdk-for-net?source=orgpage)
+- Ask a question on [Twitter](https://twitter.com/AzureSDK?s=20)
```
Comment (jsquire) [inline support.md:13]:
```suggestion
- Ask a question on [Twitter](https://twitter.com/AzureSDK)
```

File: support.md (hunk 4 above)
Comment (weshaggard) [inline support.md:1]: Can we please change the casing of this file to be `SUPPORT.md` similar to the other md files in the repo.

Comment (elraikhm) [reply support.md]: Done

**Pull request merged**